redfish-core/lib/hypervisor_system.hpp

```diff
@@ -121,7 +128,7 @@ inline void
             return;
         }
 
-        if (objInfo.empty())
+        if (objInfo.size() == 0)
```

Comment:
objInfo is a vector, so it is better to use empty() to check the size. This will keep the code as close to upstream as possible...